Build/create-sbom.ps1

```diff
@@ -0,0 +1,97 @@
+﻿param(
+    [string] $TempFolder = "Build\BuildOutput\temp\sbom",
+    [Parameter(mandatory=$true)] [string] $NpmProjectJson,
+    [Parameter(mandatory=$true)] [string] $Solution,
+    [Parameter(mandatory=$true)] [string] $GithubUsername,
+    [Parameter(mandatory=$true)] [string] $GithubAccessToken,
+    [Parameter(mandatory=$true)] [string] $OutputPath
```

Comment:
Normally, the file name including version would be provided by build script. now with the tacked-on implementation, we can pass the version and the outputDirectory.

for localbuilds we use 0.0.0 as version and probably THIS as the directory like with the other paths and append "Build\BuildOutput"

In both cases we then append "SBOM" and the filename.

Reply:
Why do we need to include the version info here? Right now, this is a problem that the caller has to solve and I think it is better this way instead of concatenating in the script. On TC it should be easy to put the version into the output file name. The script itself is not integrated into the local build so in the rare case we need to do a local built you need to pass the values yoursef anyway.